core/perf_test/CMakeLists.txt

```diff
@@ -28,15 +28,21 @@ find_package(benchmark QUIET 1.5.6)
 if(benchmark_FOUND)
   message(STATUS "Using google benchmark found in ${benchmark_DIR}")
 else()
-  message(STATUS "No installed google benchmark found, fetching from GitHub")
   include(FetchContent)
   set(BENCHMARK_ENABLE_TESTING OFF)
 
   list(APPEND CMAKE_MESSAGE_INDENT "[benchmark] ")
+
+  if (EXISTS ${CMAKE_SOURCE_DIR}/tpls/googlebenchmark-v1.7.1.tar.gz)
+    set(BENCHMARK_SRC_URL ${CMAKE_SOURCE_DIR}/tpls/googlebenchmark-v1.7.1.tar.gz)
+  else()
+    set(BENCHMARK_SRC_URL https://github.com/google/benchmark/archive/refs/tags/v1.7.1.tar.gz)
+  endif()
```

Comment:
what is this? It looks like you added locally a tar ball for google benchmark to Kokkos?

Reply:
yes, this is a small trick to avoid always (re)downloading from the internet each time.

Note: these test-related changes are put in a separate commit from the commit of SVE backend.

Reply:
This is already in another commit (2 commits in this PRs). Should it be in another PR instead ?